### PR TITLE
Enable all command line options to be provided by the .dojorc config file

### LINKED ISSUE #210: Use args from `dojorc` as defaults
**Enhancement** 

The cli should pull default args from the dojorc config by default when running a command. 
At present this is implemented as custom code in a command but should be moved into `cli`.

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [x] Unit or Functional tests are included in the PR

**Description:**

Enable all command line options to be specified via the `.dojorc` but still overridable by the command line when explicitly specified.

Order of precedence:

1) Explicitly specified command line options
2) `.dojorc` configuration
3) Default value for command line options 

Resolves #210

### CODE REVIEW

**Comment by agubler** [comment]
@tomdye yup good point I'll add a section in the readme

**Comment by agubler** [comment]
@tomdye added a section in the readme on the dojorc.

File: README.md [hunk 1]
```diff
@@ -92,6 +92,35 @@ The CLI has the following built-in groups:
 `npm i @dojo/cli-build-webpack` and `npm i @dojo/cli-test-intern`
 These two groups are not included by default to allow different versions of these groups to be installed per project.
 
+## dojorc
```
Comment (agubler) [inline README.md]: @dylans do you mind reviewing this?

File: README.md [hunk 2]
```diff
@@ -92,6 +92,35 @@ The CLI has the following built-in groups:
...
+
+```json
+{
+	"build-app": {
+		"build-app-option": "foo"
+	}
+}
+```
+
+This configuration would automatically pass the `build-app-option: foo` to the command but can be overridden by passing the option on the command line:
```
Comment (dylans) [inline README.md]: but can be overridden -> but is overridden

**Pull request merged**